tools/certa/src/runners/chrome/ChromeTestRunner.ts

```diff
@@ -65,6 +65,7 @@ export class ChromeTestRunner {
       writeCoverageData(coverage);
 
     process.exitCode = failures;
+    (process as any).emit("chrome-test-runner-done");
```

Comment:
Why does this work but starting a timer when writing the test epilogue doesn't?

I mean, I know doing the handle leak detection in our reporter is pretty hacky in the first place, but I'm confused why we'd get to here 10s of seconds after the tests finished.  Is chrome really taking that long to shut down?  And if so, how do we know that we're not ultimately causing that with a handle leak?

Reply:
I have no idea why the timer didn't work from epilog, but I got consistent failure when running on macOS. (As an aside, the timer had recently been bumped to 30 seconds, and I dropped it back to 10 after moving the timeout.) My understanding is that multiple modules are being tested concurrently during `rush cover`, and that seems to contribute, as I don't think I ever saw the failure when just testing one module. But I still have no idea why it takes so long.

Reply:
OK, I still think this warrants more investigation, but I understand this will help us unblock builds.

Let's merge this PR but keep #5017 open until we understand the root cause here..